.gitignore

```diff
@@ -21,3 +21,5 @@ cabal.project.local
 cabal.project.local~
 .HTF/
 .ghc.environment.*
+*~
+src/copy/
```

Comment:
What's this for?

Reply:
there was a copy of the project because I hadn't committed a few important changes but I don't need now

Reply:
So you should remove this from the `.gitignore`